server/chat.ts

```diff
@@ -1006,8 +1007,14 @@ export class CommandContext extends MessageContext {
 			// and these PM-related messages aren't attached to any rooms. If we ever get to letting users set their
 			// own language these messages should also be translated. - Asheviere
 			if (targetUser) {
-				if (lockType && !targetUser.can('lock')) {
-					this.errorReply(`You are ${lockType} and can only private message members of the global moderation team. ${lockExpiration}`);
+				if (
+					lockType &&
+					!targetUser.can('lock') &&
+					!(canPMRoomOwners && [...Rooms.rooms.values()].some(curRoom => (
+						!curRoom.settings.isPersonal && targetUser?.can('editroom', null, curRoom)
+					)))
```

Comment:
Oh, wait, this is one direction. The RO won't be able to reply since you haven't also edited:

![image](https://user-images.githubusercontent.com/551184/90322137-4beee780-df05-11ea-9982-77f44e29e3af.png)

I think you probably want a function for "can PM locked users" so this isn't duplicated.

Reply:
Oh. I'm not sure how I missed that. Anyway, it's fixed now.